packages/datadog-instrumentations/src/next.js

```diff
@@ -147,18 +199,59 @@ function finish (ctx, result, err) {
   return result
 }
 
+// transpiled functions in newer next.js versions can't be shimmed
+// because they're read-only, so we re-define the property needed here
+function wrapObjectProperty (module, func, wrapper) {
```

Comment:
Not 100% sure on the syntax here, but I wasn't able to shim the functions in these files because they trans-piled as "read-only", so I tried modifying Object properties. Can change around as needed!